index.html

```diff
@@ -226,7 +250,7 @@ <h4>Skills</h4>
       </div>
     </section>
 
-    <footer class="flex footer">
+    <footer id="contact" class="flex footer">
       <div class="flex footer-wrapper">
         <h2 class="section-title">Contact me</h2>
         <p class="section-description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
```

Comment:
- [ ] Please reduce the gap between the contact section paragraph and the form and make sure it matches the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/165697540-b19bb37a-3829-474c-b2a1-d957c0daf020.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/165697591-ca1604bc-165e-49b3-90d6-851e6614a595.png)